recipes/kennard-stone/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "kennard-stone" %}
+{% set version = "1.1.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://files.pythonhosted.org/packages/7a/ce/a4e3b09323ed9e7ae0054795927ba62d1d9bfc5d3e09f0d92b8706a50e06/kennard_stone-1.1.1.tar.gz"
```

Comment:
This should use the predictable form of the URL templated with `{{ version }}`. See `recipes/example/meta.yaml` for an example.